text/0000-abort-by-default.md

```diff
@@ -0,0 +1,74 @@
+- Feature Name: abort_by_default
+- Start Date: 2016-10-01
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Specify `panic=abort` in `Cargo.toml` when the user does `cargo new --bin`.
+
+# Motivation
+[motivation]: #motivation
+
+## Performance
+
+Generally, the performance of bigger programs [improves by 10%](https://www.youtube.com/watch?v=mRGb4hoGuPs), which is no small amount. When overflow checking is enabled, it is as high as 2x, making overflowing checking plausible in production.
+
+The performance gains mainly originates in the fact that it is a lot easier for the compiler to optimize, due to the unwind path disappearing, and hence reasoning about the code becomes easier.
+
+## Binary size
+
+Binary size improves by 10% as well. This is due to the lack of landing pads which are injected in the stack when unwinding is enabled.
+
+## Compile time
+
+Compile time in debug mode improves by 20% on average. In release mode, the number is around 10%.
+
+## Runtime size
+
+Unwinding requires a fair amount of runtime code, which can be seen as conflicting with the goals of Rust.
+
+## Correctness
+
+You often see people abusing `std::panic::catch_unwind` for exception handling. Forcing the user to explicitly opt in to unwinding will make it harder to unintentionally misuse it.
+
+# Detailed design
+[design]: #detailed-design
+
+Have `cargo` generate `panic=abort` to the `Cargo.toml`. Whenever the user inteacts with `cargo` (by using a command) and the `Cargo.toml` has no specified panicking strategy, it will add `panic=unwind` and leave a note to the user: 
+
+    Warning: No panic strategy was specified, added unwinding to the `Cargo.toml` file, please consider change it to your needs.
+
+For libraries, nothing is modified, as they inherit the strategy of the binary.
```

Comment:
What happens for crates with both? I believe this is a fairly common pattern; I assume they would set it to unwind since they have the binary component?